sig-contributor-experience/meeting-notes-archive/2020-meeting-notes.md

```diff
@@ -1852,7 +1852,7 @@ Agenda:
             * [Some docs need to be updated](https://github.com/kubernetes/community/issues/4904)
         * [https://hackmd.io/@dz39yKwDRheGNhkOnAiykw/r1wNy_86U](https://hackmd.io/@dz39yKwDRheGNhkOnAiykw/r1wNy_86U) - template from rajula on marketing team 
     * Marketing update [paris]
-        * Need more storytellers [https://github.com/kubernetes/community/blob/master/communication/marketing-team/role-handbooks/storytellers.md](https://github.com/kubernetes/community/blob/master/communication/marketing-team/role-handbooks/storytellers.md)
+        * Need more storytellers [https://github.com/kubernetes/community/blob/master/communication/contributor-comms/role-handbooks/storytellers.md](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/role-handbooks/storytellers.md)
```

Comment:
Same here, we can leave this file alone.

Reply:
yep